Userland/Libraries/LibGUI/Clipboard.cpp

```diff
@@ -57,14 +57,14 @@ Clipboard& Clipboard::the()
 Clipboard::DataAndType Clipboard::fetch_data_and_type() const
 {
     auto response = connection().get_clipboard_data();
+    auto type = response.mime_type();
+    auto metadata = response.metadata().entries();
     if (!response.data().is_valid())
```

Comment:
this should probably now read
```suggestion
    if (response.data().is_empty())
```
is it possible to change that upstream? 🤔

Reply:
I'm not sure `is_empty()` is better in this case, as we are not checking whether `data()` is empty, but `null`